src/components/ArrowToggle/README.md

```diff
@@ -4,11 +4,11 @@
 
 <!--/GITHUB_BLOCK-->
 
-`ArrowToggle` is a component that displaying chevron icon. It can rotate in 4 directions. It could be used for displaying dropdown lists or cut components, etc.
+`ArrowToggle` is a component for displaying the chevron icon. It be rotated to point in four directions, and can be usef to display dropdown lists, cut components, etc.
```

Comment:
`It be rotated to point` sounds a little bit weird

Reply:
Thank you. Reverted it to the old variant.